public/locales/en/translation.json

```
@@ -622,12 +622,32 @@
             "general": "General",
             "log": "Log",
             "other": "Other",
-            "sync": "Cloud Saves Sync"
+            "sync": "Cloud Saves Sync",
+            "systemInformation": "System Information"
         },
         "open-config-file": "Open Config File",
         "reset-heroic": "Reset Heroic",
         "saves": {
             "warning": "Cloud Saves feature is in Beta, please backup your saves before syncing (in case something goes wrong)"
+        },
+        "systemInformation": {
+            "copyToClipboard": "Copy to clipboard",
+            "CPU": "CPU:",
+            "gogdlVersion": "Gogdl: {{gogdl}}",
+            "gpu": "GPU:",
+            "gpuWithNumber": "GPU {{no}}:",
+            "heroicVersion": "Heroic: {{heroic}}",
+            "legendaryVersion": "Legendary: {{legendary}}",
+            "memory": "Memory:",
+            "memoryStats": "{{percentUsed}}% used ({{usedGib}} / {{totalGib}}",
+            "nileVersion": "Nile: {{nile}}",
+            "os": "Software:",
```

```suggestion
            "os": "Operating System:",
```

based on https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/3027/files#diff-15995a3ce6795c647cff923de4a7ae8adbacc9d0c170c74425cb37e319e1e134R37